docker-compose.yml

```diff
@@ -9,7 +12,7 @@ db:
 web:
   #image: kartoza/osm-reporter
   build: .
-  dockerfile: Dockerfile
+  dockerfile: Dockerfile.dev
```

Comment:
Do you mean the not dev dockerfile is not used anymore?

Reply:
The dev docker would be used for local development and the Dockerfile for pushing images to docker hub for use by the catalogue which cannot do local builds.